test/support/test_util.ex

```diff
@@ -88,6 +90,9 @@ defmodule Flop.TestUtil do
   defp get_field(pet, {:join, %{path: [a, b]}}),
     do: pet |> Map.fetch!(a) |> Map.fetch!(b)
 
+  defp matches?(:empty, v, {:array, _}), do: &(v == Enum.empty?(&1))
+  defp matches?(:not_empty, v, {:array, _}), do: &(v == !Enum.empty?(&1))
+  defp matches?(op, v, _), do: matches?(op, v)
```

Comment:
I think we want to handle an array field as empty if it is `[]` _or_ `nil`.

Reply:
Yes, good point. I missed that.